Makefile

```diff
@@ -44,7 +44,7 @@ $(libtermbox):
 	cd vendor/termbox && ./waf configure && ./waf --targets=termbox_static
 
 update_deps:
-	@git submodule update --init --depth 1
+	@git submodule update --init --remote --recursive
```

Comment:
when i ran this in the travis debug docker container i got `fatal: dumb http transport does not support shallow capabilities` changing to this fixed it though it looks like travis has some of its own behavior around clone depth and submodules
https://docs.travis-ci.com/user/customizing-the-build/#git-clone-depth
https://docs.travis-ci.com/user/customizing-the-build/#git-submodules